### PR TITLE
Initial Import of CAF Security MVP

### PR DESCRIPTION
**Pull request by ckittel**
Initial import from all of the work that @pabrus did on the CAF Security MVP.

Anything you'd like to see changed before we merge, @dominicallen?

### CODE REVIEW

**Review by pabrus — APPROVED** [review]

File: secure/security-mvp/deploy/caf-secure-ui.json [hunk 1]
```diff
@@ -0,0 +1,700 @@
...
+                "name": "Cloud Adoption Framework - Security MVP"
+            },
+            {
+                "name": "infoBox1",
+                "type": "Microsoft.Common.InfoBox",
+                "visible": true,
+                "options": {
+                    "icon": "Info",
+                    "text": "The Microsoft Cloud Adoption Framework (CAF) for Azure is proven guidance that's designed to help create and implement the business and technology strategies necessary for organizations to succeed in the cloud. The Secure methodology provides security guidance by providing clarity for the processes, best practices, models, and experiences. This guidance is based on the lessons learned and real world experiences of real customers, Microsoft's security journey, and work with organizations like NIST, The Open Group, and the Center for Internet Security (CIS).",
+                    "uri": "https://github.com/mspnp/caf-secure-amp-infra"
```
[the remark this answers is not in the document]
Comment (ckittel) [reply secure/security-mvp/deploy/caf-secure-ui.json]: Updated.

File: secure/security-mvp/README.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+- Azure Security Center
+  - Azure Defender
+  - Azure Security Benchmark
+  - Azure CIS Regulatory reporting
+- Azure Policy
+- Hub-spoke Network Topology
+
+Use the following link to deploy it to a greenfield subscription:
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2FCloudAdoptionFramework%2Fmain%2Fsecure%2Fsecurity-mvp%2Fdeploy%2Fcaf-secure-deploy.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2FCloudAdoptionFramework%2Fmain%2Fsecure%2Fsecurity-mvp%2Fdeploy%2Fcaf-secure-ui.json)
```
[the remark this answers is not in the document]
Comment (ckittel) [reply secure/security-mvp/README.md]: Updated.

File: secure/security-mvp/deploy/deploy.ps1 [hunk 3]
```diff
@@ -0,0 +1,13 @@
+$templateRootUri="https://raw.githubusercontent.com/microsoft/CloudAdoptionFramework/main/secure/security-mvp/deploy"
```
[the remark this answers is not in the document]
Comment (ckittel) [reply secure/security-mvp/deploy/deploy.ps1]: Updated.

**Comment by ckittel** [comment]
@dominicallen gave approval via Teams, merging now.

**Pull request merged**